<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QToolButton" name="toolButton_start">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>开始游戏</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_quit">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>740</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_present">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>660</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发言</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_show">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>270</width>
     <height>500</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_presentover">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>710</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发言结束</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_voteover">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>760</y>
     <width>90</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文彩云</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>投票结束</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_information">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>111</width>
     <height>650</height>
    </rect>
   </property>
   <property name="title">
    <string>投票</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>玩家1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>玩家2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>玩家3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>玩家4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="text">
       <string>玩家5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="text">
       <string>玩家6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="text">
       <string>玩家7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="text">
       <string>玩家8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="text">
       <string>玩家9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_10">
      <property name="text">
       <string>玩家10</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_11">
      <property name="text">
       <string>玩家11</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_12">
      <property name="text">
       <string>玩家12</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_13">
      <property name="text">
       <string>玩家13</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_14">
      <property name="text">
       <string>玩家14</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_15">
      <property name="text">
       <string>玩家15</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_16">
      <property name="text">
       <string>玩家16</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_17">
      <property name="text">
       <string>玩家17</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_18">
      <property name="text">
       <string>玩家18</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_19">
      <property name="text">
       <string>玩家19</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_20">
      <property name="text">
       <string>玩家20</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
